### ISSUE TITLE
ensure correct string encoding

### ISSUE THREAD

**Issue by iros** [learntextvis/textkit#19]
Output like 

```
hello,4
my,land,5
```

Should be

```
hello,4
my\,land,5
```

**Comment by vlandham** [comment]
this is an issue for `count` and will be a problem for POS tagger output.

I believe most CSV encodings have a slightly different solution for this, where double quotes are used explicitly.

```
hello,4
"my,land",5
```

**Comment by vlandham** [comment]
Here is ruby's:

```ruby
require 'csv'
v = ["test,a",3]
v.to_csv
=> "\"test,a\",3\n"
```

**Comment by vlandham** [comment]
Here is python



```python
import csv
v = ["test,a",3]
with open('some.csv', 'w', newline='') as f:
    writer = csv.writer(f)
    writer.writerow(v)
```

produces `"test,a",3` - so its the same.

**Comment by vlandham** [comment]
I think we could just use the python `csv.writer()` in the `count` output and the POS output, and then use `csv.reader` in some of the package functions.

**Comment by vlandham** [comment]
i can work towards doing this for `counts`

**Comment by vlandham** [comment]
Addressed in #24
